kubernetes/loculus/templates/_common-metadata.tpl

```diff
@@ -350,15 +337,29 @@ fields:
 {{- end}}
 
 {{- define "loculus.publicRuntimeConfig" }}
-            {{- if $.Values.codespaceName }}
-            "backendUrl": "https://{{ .Values.codespaceName }}-8079.app.github.dev",
+{{- $publicRuntimeConfig := (($.Values.website).runtimeConfig).public }}
```

Comment:
Could you explain the nested parens to me?

Reply:
https://stackoverflow.com/questions/59795596/how-to-make-nested-variables-optional-in-helm

It makes everything optional. It's the equivalent of what's `?.` in Typescript (there it would be `Values?.website?.runtimeConfig?.public`).

Reply:
Thanks!